src/components/pagination/bl-pagination.stories.mdx

```diff
@@ -0,0 +1,74 @@
+import { html } from 'lit';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { styleMap } from 'lit/directives/style-map.js';
+import { unsafeHTML } from 'lit/directives/unsafe-html.js';
+import { Meta, Canvas, ArgsTable, Story, Preview, Source } from '@storybook/addon-docs';
+
+<Meta
+  title="Components/Pagination"
+  component="bl-pagination"
+/>
+
+export const SimplePagination = (args) => html`
+      <bl-pagination current-page=1 total-items=1500 items-per-page=10></bl-pagination>
+  `
```

Comment:
User args to set property values via Story parameters. So Storybook canvas can be used as a playground.